docs/guides/nrfconnect_factory_data_configuration.md

```diff
@@ -0,0 +1,503 @@
+|``product_id``|product ID|2 B|uint16|mandatory|A unique ID assigned by the device vendor to identify the product. Defaults to a CSA-assigned ID that designates a non-production or test product.|
+|``vendor_name``|vendor name|<1, 32> B|ASCII string|mandatory|A human-readable vendor name that provides a simple string containing identification of device's vendor for the application and Matter stack purposes.|
+|``product_name``|product name|<1, 32> B|ASCII string|mandatory|A human-readable product name that provides a simple string containing identification of the product for the application and the Matter stack purposes.|
+|``date``|manufacturing date|<8, 10> B|ISO 8601|mandatory|A manufacturing date specifies the date that the device was manufactured. The date format used is ISO 8601, for example ``YYYY-MM-DD``.|
+|``hw_ver``|hardware version|2 B|uint16|mandatory|A hardware version number that specifies the version number of the hardware of the device. The value meaning and the versioning scheme is defined by the vendor.|
+|``hw_ver_str``|hardware version string|<1, 64> B|uint16|mandatory|A hardware version string parameter that specifies the version of the hardware of the device as a more user-friendly value than that presented by the hardware version integer value. The value meaning and the versioning scheme is defined by the vendor.|
+|``rd_uid``|rotating device ID unique ID|<16, 32> B|byte string|mandatory|The unique ID for rotating device ID, which consists of a randomly-generated 128-bit (or longer) octet string. This parameter should be protected against reading or writing over-the-air after initial introduction into the device, and stay fixed during the lifetime of the device.|
+|``dac_cert``|(DAC) Device Attestation Certificate|<1, 602> B|byte string|mandatory|The Device Attestation Certificate (DAC) and the corresponding private key are unique to each Matter device. The DAC is used for the Device Attestation process and to perform commissioning into a fabric. The DAC is a DER-encoded X.509v3-compliant certificate, as defined in RFC 5280.|
+|``dac_key``|DAC private key|68 B|byte string|mandatory|The private key associated with the Device Attestation Certificate (DAC). This key should be encrypted and maximum security should be guaranteed while generating and providing it to factory data.|
+|``pai_cert``|Product Attestation Intermediate|<1, 602> B|byte string|mandatory|An intermediate certificate is an X.509 certificate, which has been signed by the root certificate. The last intermediate certificate in a chain is used to sign the leaf (the Matter device) certificate. The PAI is a DER-encoded X.509v3-compliant certificate as defined in RFC 5280.||
+|``spake2_it``|SPAKE2 iteration counter|4 B|uint32|mandatory|The SPAKE2 iteration counter is associated with the ephemeral PAKE passcode verifier to be used for the commissioning. The iteration counter is used as a crypto parameter to process the SPAKE2 verifier.|
```

Comment:
```suggestion
|``spake2_it``|SPAKE2 iteration count|4 B|uint32|mandatory|The SPAKE2 iteration count is associated with the ephemeral PAKE passcode verifier to be used for the commissioning. The iteration count is used as a crypto parameter to generate the SPAKE2 verifier.|
```
Also, I'm not sure I understand the part about ephemeral PAKE passcode. The passcode is fixed per device, so the word "ephemeral" may be misleading.

Reply:
Ok, I see. I took it from the spec, but there is no context, so the word "ephemeral" is misleading.